### PR TITLE
[Notification] Start notification with command-line interface

### PR DESCRIPTION
**Pull request by jiangxin369**
## What is the purpose of the change

Make notification scripts using cli.

## Brief change log

*(for example:)*
  - Make notification scripts using cli.

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

### CODE REVIEW

**Review by Sxnan — APPROVED** [review]
LGTM

File: lib/notification_service/notification_service/cli/commands/server_command.py [hunk 1]
```diff
@@ -48,6 +48,9 @@ def make_log_dir_if_not_exist():
 def server_start(args):
     pid_file_path = os.path.join(notification_service.settings.NOTIFICATION_HOME,
                                  notification_service.settings.NOTIFICATION_PID_FILENAME)
+    if os.path.exists(pid_file_path):
```
Comment (Sxnan) [inline lib/notification_service/notification_service/cli/commands/server_command.py:51]: We can use `check_pid_exist` method to check if the process is actually running

**Review by Sxnan — CHANGES_REQUESTED** [review]

File: lib/notification_service/notification_service/cli/commands/server_command.py (hunk 1 above)
Comment (jiangxin369) [reply lib/notification_service/notification_service/cli/commands/server_command.py:51]: Thanks for your review, I'll add a check.

**Review by Sxnan — APPROVED** [review]
LGTM

**Pull request merged**